### PR TITLE
Port schema validation for error analysis to responsibleai

### PR DESCRIPTION
**Pull request by gaugup**

### CODE REVIEW

File: responsibleai/responsibleai/_managers/error_analysis_manager.py [hunk 1]
```diff
@@ -273,6 +281,15 @@ def get(self):
         """
         return self._ea_report_list
 
+    def _get_error_analysis_schema(self):
+        """Get the schema for validating the error analysis output."""
+        schema_directory = (Path(__file__).parent.parent / '_tools' /
```
Comment (imatiach-msft) [inline responsibleai/responsibleai/_managers/error_analysis_manager.py:286]: should these strings be moved to some constants file?

**Review by imatiach-msft — APPROVED** [review]

**Comment by imatiach-msft** [comment]
FYI the schema validations file has changed (and I have a PR to update it) to include the importance values calculated via mutual information for the features in the tree view

File: responsibleai/responsibleai/_managers/error_analysis_manager.py (hunk 1 above)
Comment (gaugup) [reply responsibleai/responsibleai/_managers/error_analysis_manager.py:286]: I see the same pattern in other files as well. https://github.com/microsoft/responsible-ai-widgets/blob/212dc5a168a196ab71ea7a29b60b0751ca301509/responsibleai/responsibleai/_managers/counterfactual_manager.py#L378. Will probably keep like others for now.

**Comment by gaugup** [comment]
> FYI the schema validations file has changed (and I have a PR to update it) to include the importance values calculated via mutual information for the features in the tree view

The tests seem to pass with the current schema. Are we having the test coverage for the new schema?

**Pull request merged**